pkg/flowkit/config/json/network_test.go

```diff
@@ -47,13 +47,10 @@ func Test_ConfigNetworkSimple(t *testing.T) {
 
 func Test_ConfigNetworkMultiple(t *testing.T) {
 	b := []byte(`{
-    		"emulator": {
-				"host": "127.0.0.1:3569",
-				"network-key": ""
-			},
+    		"emulator": "127.0.0.1:3569",
     		"testnet": {
-				"host": "access.testnet.nodes.onflow.org:9000",
-				"network-key": ""
+				"Host": "access.testnet.nodes.onflow.org:9000",
+				"NetworkKey": "5000676131ad3e22d853a3f75a5b5d0db4236d08dd6612e2baad771014b5266a242bccecc3522ff7207ac357dbe4f225c709d9b273ac484fed5d13976a39bdcd"
```

Comment:
We should still use lowercase field names to be consistent with the rest of the config (can use eg. `json:"host"` struct tags for this).

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/2e35af800a3cefd8b912866f96951c5da2e4a326